.ahoy/site/.scripts/site.files-fix-permissions.sh

```diff
@@ -8,9 +8,7 @@ if [ -d docroot/sites/default/files ]; then
   find docroot/sites/default/files/ -type f -exec chmod o+rw {} \;
 fi
 
-mkdir -p docroot/sites/default/private
-
 if [ -d docroot/sites/default/private]; then
```

Comment:
The if statement fails without space before close bracket: `if [ -d docroot/sites/default/private ]; then`